packages/core/package.json

```diff
@@ -16,8 +16,6 @@
     "access": "public"
   },
   "files": [
-    "v4.d.ts",
```

Comment:
Is this intended?

Reply:
I deleted the real files in a previous PR without touching the `package.json`. So yes.